vllm/worker/hpu_model_runner.py

```diff
@@ -1475,7 +1502,8 @@ def warmup_model(self, kv_caches: List[torch.Tensor]) -> None:
         max_blocks = kv_caches[0][0].size(0)
 
         self.prompt_buckets, prompt_omitted_buckets = generate_prompt_buckets(
```

Comment:
Shouldn't be `self.bucketing_global_state.prompt_buckets` here?

Reply:
should be, thanks